pom.xml

```diff
@@ -71,6 +71,23 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.commonjava.maven.plugins</groupId>
+        <artifactId>directory-maven-plugin</artifactId>
```

Comment:
We need to wait for the CQ, see also https://github.com/eclipse/smarthome/pull/4240